generator/altitude_generator.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "std/string.hpp"
+
+namespace routing
+{
+void BuildRoadAltitudes(string const & srtmPath, string const & baseDir,
```

Comment:
Also, you will be able to make SrtmTileManager virtual with two subclasses - the first one will work with real srtm files, the second one will be used in testing.
After that, testing of serialization - deserialization will be trivial.